### DISCUSSION TITLE
Where to put java.lua file

### DISCUSSION THREAD

**Discussion by Ralghs** [LunarVim/LunarVim#3859] [Q&A] [answered]
I am trying to set up jdtls and I see most tutorials tell me to put my java.lua file under ~/.config/nvim(in this case lvim)/ftplugin but I see a ftplugin file already in ~/.local/share/lunarvim/lvim. Does it matter where I put my java.lua file? Can I just choose any? What is the difference

**Comment by LostInTheLogs** [comment, marked as answer]
don't edit files in `~/.local/share/lunarvim/*`, put it in .cofing/lvim/ftplugin/java.lua

**Comment by Fr3027** [comment]
Jdtls seems not pick java.lua at ~/.config/lvim/ftplugin, I set up Lombok support in java.lua, but the jdtls continue saying: can't find symbol ,please help me out.

**Reply by Ralghs** [reply]
There might be an error in your java.lua file. I think it would be best to open a new issue. Remember to upload your setup such as lsp logs and java.lua file so its easier for them to help you
